dashdotdb/__init__.py

```diff
@@ -10,11 +10,11 @@
 from dashdotdb.models import deploymentvalidation  # type: ignore  # noqa: F401
 
 
-DATABASE_HOST = os.environ['DATABASE_HOST']
-DATABASE_PORT = os.environ['DATABASE_PORT']
-DATABASE_USERNAME = os.environ['DATABASE_USERNAME']
-DATABASE_PASSWORD = os.environ['DATABASE_PASSWORD']
-DATABASE_NAME = os.environ['DATABASE_NAME']
+DATABASE_HOST = os.getenv('DATABASE_HOST') or 'localhost'
+DATABASE_PORT = os.getenv('DATABASE_PORT') or '5432'
+DATABASE_USERNAME = os.getenv('DATABASE_USERNAME') or 'postgres'
+DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD') or 'postgres'
+DATABASE_NAME = os.getenv('DATABASE_NAME') or 'postgres'
```

Comment:
this needs rebase :)

Reply:
ah-yup. TODO :)

Reply:
i dont understand what happened. all of my recent changes since several days ago /and/ new files after the rebase were completely and totally wiped out.. and i force pushed the post-rebase commits to my fork.[ screaming intensifies ].. and apparently spideroak is no longer installed on my system.. super thrilled.

Reply:
TIL: with the events api, as long as i pushed the code up to github, i can revert back to an on-github stored sha even after a force push.